sdk/communication/communication-common/src/index.ts

```diff
@@ -6,8 +6,8 @@ export {
   CommunicationGetTokenOptions,
 } from "./communicationTokenCredential";
 export { AzureCommunicationTokenCredential } from "./azureCommunicationTokenCredential";
-export * from "./credential";
 export { CommunicationTokenRefreshOptions } from "./autoRefreshTokenCredential";
+export { EntraTokenCredential, EntraCommunicationTokenCredentialOptions } from "./entraTokenCredential";
```

Comment:
EntraTokenCredential shouldn't be exported

Reply:
fixed